perception/image_projection_based_fusion/config/pointpainting.param.yaml

```diff
@@ -1,10 +1,14 @@
 /**:
   ros__parameters:
     class_names: ["CAR", "PEDESTRIAN", "BICYCLE"]
+    rename_car_to_truck_and_bus: true
     point_feature_size: 7  # x, y, z, timelag and car, pedestrian, bicycle
     max_voxel_size: 40000
     point_cloud_range: [-76.8, -76.8, -3.0, 76.8, 76.8, 5.0]
     voxel_size: [0.32, 0.32, 8.0]
     downsample_factor: 1
     encoder_in_feature_size: 12
     yaw_norm_thresholds: [0.3, 0.0, 0.3]
+    input_offset_ms: [61.67, 111.67, 45.0, 28.33, 78.33, 95.0]
+    timeout_sec: 0.07
+    match_threshold_ms: 50
```

Comment:
```suggestion
```
They are in another param file.

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/2175/files#diff-d7a2406737247406518fdab67366a31dad5e5f8ddf94f13e661e4c22d8ee0633